src/Commands/Install.php

```diff
@@ -73,7 +73,7 @@ public function handle()
         $this->info('✔️  Created routes/tenant.php');
 
         $this->line('');
-        $this->line("This package lets you store data about tenants either in Redis or in a relational database like MySQL. If you're going to use the database storage, you need to create a tenants table.");
+        $this->line("This package lets you store data about tenants either in Redis or in a relational database like MySQL. If you're going to use the database storage, you need to create tables for tenants and domains.");
```

Comment:
perhaps `for tenants and domains` should be `for tracking tenants and domains`?

I know the tables that the migrations will create are literally called `tenants` and `domains`, but when running `install` the new user of the package won't understand that yet, so adding the word `tracking` (or "storing"?) may add clarity about purpose.

Reply:
I originally wanted to go with something like `you need to create a tenants table and a domains table` but that felt repetitive and I couldn't find anything that sounded better. Any idea how to shorten that sentence? :smile:

Reply:
Sure: "You need migrations. Say yes to install them."  :D 

Okay... Second sentence, this should suffice:
`For database storage we have provided some migrations.`

Reply:
Maybe something along the lines of:
> To store data about tenants in a relational database, you need a few database tables. Do you wish to publish the migrations that create these tables?

Reply:
Yes, that's fair.